[MRG] Disclaimer, Imprint FR (#94)
BlockSigner/cove: site/content/english/sav-fsa.md

{{< side-by-side >}}
{{% content %}}

> ## The qualified electronic signature is made simple, intuitive and efficient with Skribble.

Reviewer: ```suggestion
## The qualified electronic signature is simple, intuitive and efficient with Skribble.
```
I'd drop the "made", it sounds a bit weird. If we want to keep it I think we should go for "is made simple, blah, .. by Skribble". I think it is "macbooks are made by apple" and "macbooks are made with aluminium".

{{% fancy-list %}}
- **No external hardware required** Skribble can be used immediately

> - **Attach multiple signatures** Sign multiple documents with one click (batch signature)

Reviewer: When I started reading this bullet point I thought "ah, great multiple people can sign a document!" but the end of the sentence is about me signing several documents at once/with one click. This is also what the German sentence is about. I don't have a good suggestion for what to replace "attach multiple signatures" with. Some ideas: "Multi signature", "many signature", "batch signature", "Autopen signature" ([autopen](https://en.wikipedia.org/wiki/Autopen)). Not a fan of any of these really :-/

{{< picture image5 430 >}}

{{% content %}}
## Skribble meets the highest legal and technological standards
{{% fancy-list %}}
- **ZertEs & eIDAS** QES certificates from Swisscom

> - **DSG/ DSGVO** AES-256 Encryption

Reviewer: ```suggestion
- **DSG / DSGVO** AES-256 Encryption
```
missing space or on purpose?

> - **Schweizer Hosting** FINMA-compliant, ISO 27001 certified

Reviewer: ```suggestion
- **Swiss Hosting** FINMA-compliant, ISO 27001 certified
```

{{% center-block %}}
{{% content %}}
# SAV-FSA members use Skribble at a preferential price

{{< picture sav-special 200 >}}

## per month / user instead of CHF 25.00

> Signatures can also be obtained independently of the user.

Reviewer: I'm not sure I understand what this means, either in German or English. Do you think the reader will know what we are trying to say with this?

{{< picture identification 482 >}}

{{% content %}}
# Ready to sign in 5 minutes

All you need is a mobile phone and a valid identification document (passport or identity card).

{{% fancy-list %}}

> - **Skribble's coming by your office**

Reviewer: ```suggestion
- **Skribble visits to your office**
```

- **We will identify you**
- **You're ready to sign right away**
{{% /fancy-list %}}

> A law firm may also be empowered to identify persons itself.

Reviewer: ```suggestion
A law firm may also elect to identify persons itself.
```
or maybe "register", "sign up" or "A law firm can be enabled to identify people itself."? I think I prefer "people" over "persons" (this makes me think of persons of interest, people who are wanted for a crime ;) )

{{< picture image8 414 >}}

{{% content %}}

> # Sign now legally valid electronically

Reviewer: I really liked the "Legally binding electronic signatures" from the top. Can we reuse the "legally binding" instead of "legally valid"?

Maybe "Sign now, electronically and legally binding"?

> # Sign now legally valid electronically

Author: ```suggestion
# Sign now, legally valid electronically
```
